docs/wg/research/chromium/svg/index.md

```
@@ -51,13 +56,162 @@ These sit under `docs/wg/research/chromium/` (not this subdirectory):
 
 ## Source locations
 
-All findings are from the `third_party/blink/renderer/core/` subtree:
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)